src/main/jbake/content/docs/user/quick-start.adoc

```diff
@@ -40,6 +40,11 @@ bin/standalone.sh  # use standalone.bat on Windows
 You can then navigate your browser to http://localhost:8080/. When you see the login screen,
 you have to register as a user and will then be redirected to the main Hawkular page.
 
+WARNING: Should you decide to use another hostname and/or port, you'd need to change two files to point to the currect
+URL for Keycloak: `$HAWKULAR_HOME/modules/org/hawkular/nest/main/deployments/hawkular-console.war/keycloak.json` and
+`$HAWKULAR_HOME/standalone/configuration/standalone.xml`. Replace the instances of `http://localhost:8080` on the
```

Comment:
Would it make sense to show the exact line? Basically what you removed from external-keycloak.adoc ?